odl/space/weighting.py

```diff
@@ -498,7 +498,7 @@ def _inner_default(x1, x2):
         # This could also be done with `np.vdot`, which has complex conjugation
```

Comment:
This comment is now out of date. Fixed in d9575e856d7fcfaea09782cbba041952b3bf581c